packages/bsky/src/api/com/atproto/admin/takeModerationAction.ts

```diff
@@ -98,12 +103,30 @@ export default function (server: Server, ctx: AppContext) {
           { create: createLabelVals, negate: negateLabelVals },
         )
 
-        return result
+        return { result, takenDown }
       })
 
+      if (takenDown) {
+        const { did, subjects } = takenDown
+        if (did && subjects.length > 0) {
+          const agent = await ctx.pdsAdminAgent(did)
+          await Promise.all(
```

Comment:
Total nit, but `allSettled()` might be nice here so that if one fails we still wait for others to succeed.

Reply:
yup good point 👍 

do you think we should report an error if one does not succeed? or should we just log?

Reply:
It's a hard call, but I guess I lean a little towards reporting an error that tells the user that it was partially applied.

Reply:
cool cool 👍 

i ended up doing that with an `UpstreamFailureError`